lib/commands/update.sh

```diff
@@ -0,0 +1,33 @@
+update_command() {
+  local update_to_head=$1
+
+  (
+  cd $(asdf_dir)
+
+  if [ "$update_to_head" = "--head" ]; then
+    # Update to latest on the master branch
+    git checkout master || exit 1
+
+    # Pull down the latest changes on master
+    git pull origin master || exit 1
+    echo "Updated asdf to latest on the master branch"
+  else
+    # Update to latest release
+    git fetch --tags || exit 1
+    tags=$(git tag | sort_versions | tail -r) || exit 1
+
+    # Pick the newest tag
+    tag=$(echo $tags | cut -d ' '  -f1)
+
+    # Update
+    git checkout "$tag" || exit 1
+    echo "Updated asdf to release $tag"
+  fi
+  )
+}
+
+# stolen from https://github.com/rbenv/ruby-build/pull/631/files#diff-fdcfb8a18714b33b07529b7d02b54f1dR942
```

Comment:
I actually stole this from the irdris plugin, which stole it from ruby build :p https://github.com/vic/asdf-idris/blob/master/bin/list-all#L6